# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/ctwm/manpage.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./ctwm.man.orig	2001-12-11 16:38:52.000000000 +0100
+++ ./ctwm.man	2003-04-26 20:06:36.000000000 +0200
@@ -1,13 +1,10 @@
 .de EX		\"Begin example
-.ne 5
 .if n .sp 1
 .if t .sp .5
 .nf
-.in +.5i
 ..
 .de EE
 .fi
-.in -.5i
 .if n .sp 1
 .if t .sp .5
 ..
